generative_data_prep/utils/utils.py

```diff
@@ -282,7 +283,7 @@ def _get_walk_files_to_hash(dir: str, filter: Optional[str] = None):
         hash_file_names = [
             (
                 os.path.join(foldername, filename),
-                relative_foldername + filename.split(".")[0] + ".txt",
+                relative_foldername + filename,
```

Comment:
Wait sorry what is the point of savling both the absolute and relate path?

Reply:
ah the filename would be the name of the file not the absolute path to the file. The relative_foldername is the absolute path but then truncated so its only the relative path! Basically relative_foldername + filename should only be the relative path to where the output dir is creating it.

I originally had issues where if I saved the absolute path but if you were then to move the directory say to sambasuite it would error out when checking sha since the absolute path isn't there anymore